src/CloudEvents/CloudEvent.php

```diff
@@ -1,82 +0,0 @@
-<?php
```

Comment:
Moved to https://github.com/cloudevents/sdk-php/pull/4/files#diff-7489e1f1e0cc7562d90dd81ea209e40fd2517ad0140c331ed345b97c8d753bd1. I kept the function signatures the same.